### ISSUE TITLE
[DS4 SCP bug] Dosent detect controller

### ISSUE THREAD

**Issue by Xelbair** [Kaldaien/UnX#43]
When using SCP drivers for dual shock 4 UnX calibrate detects only the 'wireless controller' instead of virtual x360 one preventing anyone(with multiple controllers) from using it.

**Comment by Kaldaien** [comment]
Yeah, you should avoid using controller emulation software. The controller is DirectInput native.
